src/main/java/org/folio/circulation/domain/policy/RequestPolicy.java

```diff
@@ -44,4 +57,32 @@ public boolean allowsType(RequestType type) {
     log.info("allowsType:: result: false");
     return false;
   }
+
+  private static Map<RequestType, Set<UUID>> extractAllowedServicePoints(
+    JsonObject allowedServicePointsJson) {
+
+    log.debug("extractAllowedServicePoints:: parameters representation: {}",
+      allowedServicePointsJson);
+
+    Map<RequestType, Set<UUID>> allowedServicePoints = new HashMap<>();
+    if (allowedServicePointsJson != null) {
+      for (RequestType requestType : RequestType.values()) {
+        JsonArray jsonArray = allowedServicePointsJson.getJsonArray(requestType.getValue());
+        if (jsonArray != null) {
+          allowedServicePoints.put(requestType, extractServicePointIds(jsonArray));
+        }
+      }
+    }
+
+    return allowedServicePoints;
+  }
+
+  private static Set<UUID> extractServicePointIds(JsonArray jsonArray) {
```

Comment:
There is no mention of service point id in this method? Should we give it a more generic name? Or maybe even extract it into a utility class?